lib/elixir/lib/module.ex

```diff
@@ -977,20 +982,29 @@ defmodule Module do
     Kernel.Typespec.spec_to_callback(module, definition)
   end
 
+  @extra_error_msg_definitions_in "Use the Module.__info__/1 callback to get public functions and macros instead"
```

Comment:
This is used just once?

Reply:
sorry, there was something wrong with my commit history. it is fixed